tests/unit/scp_test.py

```diff
@@ -57,3 +59,18 @@ def test_get(dst_path, src_path, ssh_scp, transmit_payload):
     """Check that SCP file download works."""
     ssh_scp.get(str(src_path), str(dst_path))
     assert dst_path.read_bytes() == transmit_payload
+
+
+@pytest.fixture
+def src_path_missing(tmp_path):
+    """Return a remote path that does not exist."""
+    path = tmp_path / 'non-existing.txt'
```

Comment:
oh! that's where #621 got the misleading fixture from :)

Reply:
sorry. Likely messed some parts together when I was testing both of them.

Reply:
No problem! That was mostly a note to myself — I tend to record my discoveries, as it helps to build a fuller picture of what patches do during reviews…